elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -105,6 +105,15 @@ class PfeAccordion extends PFElement {
 
       this._observer.observe(this, { childList: true });
     });
+    if ((!this.hasAttribute("pfe-disclosure")) && ( this._allHeaders().length === 1 )) {
+      this.setAttribute("pfe-disclosure", "true");
+      this._allHeaders().forEach(function(headers) {
+        headers.setAttribute("pfe-disclosure", "true");
+      });
+      this._allPanels().forEach(function(panels) {
+        panels.setAttribute("pfe-disclosure", "true");
+      });
+    };
```

Comment:
Just noting here, this needs a mutation observer so that when the value of `pfe-disclosure` is changed on the parent `<pfe-accordion>` , it propagates down to the children `<pfe-accordion-header>` and `<pfe-accordion-panel>`

Reply:
If you update the variables on pfe-accordion and let them propagate to the children that way, you might not need the extra code. You won’t need to push disclosure to the children elements at all.

Reply:
This would require quite a bit of refactoring for things that currently do not have custom properties, such as the panel padding, the chevron position / states.